### ISSUE TITLE
Could you create a package for this Workflow for vRO 8.x

### ISSUE THREAD

**Issue by imtrinity94** [burkeazbill/postmanCollections#1]
https://www.vcoteam.info/files/ProcessPostmanCollectionUpdate.workflow

I hope you got what I meant.

Thanks
Mayank
https://www.linkedin.com/in/mayankgoyal1994/

**Comment by burkeazbill** [comment]
Yes, I understand the request. I have a number of other projects going on that prevent me from taking this on any time soon though.
